### ISSUE TITLE
Remove duplicate neurons from database

### ISSUE THREAD

**Issue by travs** [openworm/PyOpenWorm#137]
From #128 the doctest was failing, saying that we had 333 neurons instead of the expected 302.

### You can skip this digging and go to my last comment for the results

It turns out that, depending on how we get the neurons form the graph, a different set is returned.

i.e.
```python
>>> len(set(P.Neuron().load()))
333
# but
>>> len(set(P.Worm().neuron_network().neurons()))
302
```

Did some more digging, and the difference between these two sets ends up being:
```python
{u'AS01',
 u'AS02',
 u'AS03',
...
 u'VB09',
 u'VD06'}
```
<sub>[(Full list here)](http://hastebin.com/sixaqumuye.py)</sub>

Neurons that end in a number that begins with zero (ex: "VB**0**9") are duplicated.
Observe:
```python
>>> net = P.Worm().neuron_network()
>>> vb09 = net.aneuron('VB09')
>>> vb9 = net.aneuron('VB9')
# but
>>> vb9 == vb09
False
```

**Comment by travs** [comment]
Further digging, and these neurons, in addition to having different names, have different values for their properties. The "zeroless" neurons have no connection information from what I can tell, but have descriptions. The neurons with zeros have no descriptions, but do have connection information.

```python
>>> vb09.description()
>>> vb9.description()
u'Vent. cord motor neuron, innervates vent. body muscles'
>>> vb09.GJ_degree()
1
>>> vb9.GJ_degree()
0
```

Going to look into `insert_worm.py` to see why this might be the case...

**Comment by travs** [comment]
It turns out that the source of information for `insert_worm.py`'s function `upload_lineage_and_descriptions()` ([linked here](http://www.wormatlas.org/celllistsulston.htm)) uses a different naming convention than the rest of PyOpenWorm. 

Instead of AS01, they use AS1, and so on.

**Comment by slarson** [comment]
I believe this is fixed by #138
